### PR TITLE
Voting: fix a description overflow issue

### PR DESCRIPTION
**Pull request by bpierre**
Before (notice the scrollbars):

image

After:

image

### CODE REVIEW

File: apps/voting/app/src/components/VoteCard/VoteCard.js [hunk 1]
```diff
@@ -97,11 +97,18 @@ const VoteCard = ({ vote, onOpen }) => {
...
           -webkit-box-orient: vertical;
           -webkit-line-clamp: 3;
           overflow: hidden;
+
+          // overflow-wrap:anywhere and hyphens:auto are not
+          // supported yet by the latest versions of Webkit / Blink,
+          // which is why word-break:break-word has been added.
+          hyphens: auto;
+          overflow-wrap: anywhere;
+          word-break: break-word;
```
Comment (sohkai) [inline apps/voting/app/src/components/VoteCard/VoteCard.js]: Should we put this in `VoteText` instead?

Comment (bpierre) [reply apps/voting/app/src/components/VoteCard/VoteCard.js]: Yes maybe we could: `VoteText` is inline but it’s only used as a block, the structure must be coming from a need in the previous version. Changing it :+1:

Comment (bpierre) [reply apps/voting/app/src/components/VoteCard/VoteCard.js]: Done in https://github.com/aragon/aragon-apps/pull/1021/commits/e6e78b7fa843f10a1099bbaee3b9fdd4f42f3f8e

File: apps/voting/app/src/components/VoteCard/VoteCard.js [hunk 2]
```diff
@@ -92,28 +92,20 @@ const VoteCard = ({ vote, onOpen }) => {
...
+        prefix={<span css="font-weight: bold">#{voteId}: </span>}
+        text={description || metadata}
+        title={`#${voteId}: ${description || metadata}`}
         css={`
+          overflow: hidden;
           ${textStyle('body1')};
-          /* lines per font size per line height */
-          /* shorter texts align to the top */
-          height: ${10.5 * GU}px;
+          height: ${27 * 3}px; // 27px = line-height with body1
```
Comment (sohkai) [inline apps/voting/app/src/components/VoteCard/VoteCard.js]: Maybe we can add a note to the comment about `3` being the number of lines we want

**Pull request merged**